### PR TITLE
feat(board): add test for duplicate board PENPOT-253

### PR DESCRIPTION
**Pull request by tdelatorre**
# Done Definition Checks

## Description

This PR add a test for duplicate board [PENPOT-253](https://app.qase.io/case/PENPOT-253).

## How to test

- [ ] Check the code
- [ ] Update the test case in Qase (Automation Status field or steps changed)
- [ ] It complies with the test conventions
- [ ] There are no missing snapshots for win32 (x3 browsers)
- [ ] The tests run OK

## Screenshots 📸 (optional)

board

### CODE REVIEW

File: tests/composition/composition-board.spec.js [hunk 1]
```diff
@@ -820,6 +820,52 @@ mainTest.describe(() => {
...
+        await mainPage.waitForChangeIsSaved();
+      });
+
+      await mainTest.step(
+        'Duplicate the first board via right click on layer in canvas',
+        async () => {
+          const board1 = 'Board #1';
+          await mainPage.duplicateLayerViaRightClickOnCanvas(board1);
+          await mainPage.waitForChangeIsSaved();
+          await expect(layersPanelPage.sidebarLayerItem).toHaveCount(4);
```
Comment (daniel-herrero) [inline tests/composition/composition-board.spec.js:843]: I'm ok with this way of validating the shape has been duplicated. It cannot be anything but the new shape layer.

**Review by daniel-herrero — APPROVED** [review]
LGMT! 👍

File: tests/composition/composition-board.spec.js (hunk 1 above)
Comment (tdelatorre) [reply tests/composition/composition-board.spec.js:843]: Yes, better than a screenshot!
